clients/python/README.md

```diff
@@ -56,11 +56,40 @@ Owner and controller of cells in the game. As a programmer of an AI, you are a
             and a dumb AI will take over the player's cells until the player
             becomes active once again.
 
+### Cell
```

Comment:
I disagree. I don't want "RTFM" to be an important part of the competition -- I want to see cool AIs and to see teams succeed because they had interesting ideas, not because *"they read the documentation"*. They have limited time. Let's help them make the best out of that time.